fli-iam/shanoir-ng
# Shanoir-issue#893 Study card warning during study creation + import

Closes #893 

How to test:
- Create a study and verify there is a warning
- Import MR data from DICOM on a study with no study-card: verify that there is a warning
